pwd_guess.py

```diff
@@ -826,6 +830,13 @@ def __init__(self, pwd_list, config=ModelDefaults(), multi_gpu=1):
         self.model_to_save = None
         self.multi_gpu = multi_gpu
         self.pwd_list = pwd_list
+        self.cumulative_chunks = 0
+        self.min_loss_early_stopping = 999999999 #A really large number
+        self.poor_batches_early_stopping = 0
+        if config.tensorboard:
+            self.callback = TensorBoard(".")
+            self.train_log_names = []
+            self.test_log_names = []
```

Comment:
It would be good to initialize these to None in the case where tensorboard is false so that we don't get confusing-to-debug attribute undefined errors later.